### ISSUE TITLE
Less-than-useful error message when Heroku launch fails

### ISSUE THREAD

**Issue by suchow** [Dallinger/Dallinger#480]
Launch failures produce JSON decode errors rather than producing a useful error message.

## Expected Behavior
When the launch step of deploying an experiment fails, the experimenter sees a useful error message that either tells them why it fails or tells them how to find out.

## Current Behavior
When the launch step fails, the experimenter sees a JSON decode error:

```
❯❯ Launching the experiment on MTurk...
2017-02-20 23:20:05,104 - requests.packages.urllib3.connectionpool - INFO - Starting new HTTPS connection (1): dlgr-5de64555.herokuapp.com
Traceback (most recent call last):
  File "/Users/monica/anaconda/envs/dallinger_env/bin/dallinger", line 11, in <module>
    load_entry_point('dallinger', 'console_scripts', 'dallinger')()
  File "/Users/monica/anaconda/envs/dallinger_env/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/Users/monica/anaconda/envs/dallinger_env/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/Users/monica/anaconda/envs/dallinger_env/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/monica/anaconda/envs/dallinger_env/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/monica/anaconda/envs/dallinger_env/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/Users/monica/Dallinger/dallinger/command_line.py", line 561, in sandbox
    deploy_sandbox_shared_setup(verbose=verbose, app=app)
  File "/Users/monica/Dallinger/dallinger/command_line.py", line 533, in deploy_sandbox_shared_setup
    launch_data = launch_request.json()
  File "/Users/monica/anaconda/envs/dallinger_env/lib/python2.7/site-packages/requests/models.py", line 741, in json
    return json.loads(self.text, **kwargs)
  File "/Users/monica/anaconda/envs/dallinger_env/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/Users/monica/anaconda/envs/dallinger_env/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/Users/monica/anaconda/envs/dallinger_env/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
```

## Steps to Reproduce (for bugs)
1. Do something that will cause an experiment to break during the launch stage of deployment
2. Observe the terminal output

## Context
Makes it harder to debug problems.

## Your Environment
I think this has existed at least since 2.7.0.

**Comment by suchow** [comment]
@alecpm Can this be closed?

**Comment by suchow** [comment]
Closed by #655.
